client/js/analysis.js

```diff
@@ -45,11 +46,16 @@ function renderingHandler(videoAnalysis) {
   renderDonutCharts(videoAnalysis.perspectiveMap);
   const totalComments = renderComments(videoAnalysis.commentArray);
   const commentsRenderTime = totalComments * COMMENT_APPEARANCE_TIME;
-  const sentiment = determineSentiment(videoAnalysis.magnitudeAndScore.magnitude,
-      videoAnalysis.magnitudeAndScore.score);
+  const sentiment = determineSentiment(
+      videoAnalysis.magnitudeAndScore.magnitude,
+      videoAnalysis.magnitudeAndScore.score
+      );
```

Comment:
nit: change to 
```
  const sentiment = determineSentiment(
      videoAnalysis.magnitudeAndScore.magnitude,
      videoAnalysis.magnitudeAndScore.score);
```

https://google.github.io/styleguide/jsguide.html#formatting-function-arguments